.github/workflows/sandbox_build_and_deploy.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         uses: TapTap21/docker-remote-deployment-action@v1.1
         with:
           remote_docker_host: ${{ secrets.SANDBOX_HOST }}
-          ssh_private_key: ${{ secrets.SSH_PRIV_KEY_GA }}
```

Comment:
Are you sure about this change?

Reply:
It's probably a bug after merging with he feature branch. The current version is the same as on the master branch.